python/pyxet/pyxet/file_system.py

```diff
@@ -582,6 +582,36 @@ def mv(self, path1, path2, *args, **kwargs):
     def move(self, path1, path2, *args, **kwargs):
         return self.mv(path1, path2, *args, **kwargs)
 
+    def add_deduplication_hints(self, path_urls):
+        """
+        Fetches and downloads all of the metadata needed for binary deduplication against 
+        all the paths given by `paths`.  Once fetched, new data will be deduplicated against 
+        any binary content given by `paths`.  
+        """
+        if isinstance(path_urls, str):
+            path_urls = [path_urls] 
+        url_paths = [parse_url(url, self.domain, partial_remote = True) for url in path_urls]
```

Comment:
Should `partial_remote` be set to `False`? Otherwise `xet://user` will be valid and that leads to errors in the underlying shards fetch logic.

Reply:
```
    If partial_remote==True, allows [repo] to be optional. i.e. it will
    parse /user or xet://user
```

Which is what we want here, as the urls being passed in may be straight up paths as this is a method off of dest_fs